.travis.yml

```diff
@@ -1,22 +1,20 @@
 language: node_js
+cache: false
```

Comment:
Good call here, we should try out removing the cache from our other repos too.

Reply:
Though I wonder if we could just delete the line instead of explicitly setting it to false.

Reply:
Looks like it doesn't hurt anything.
https://docs.travis-ci.com/user/caching/#Explicitly-disabling-caching